projects/packages/identity-crisis/src/class-exception.php

```diff
@@ -0,0 +1,13 @@
+<?php
+/**
+ * Exception class for the Identity Crisis package.
+ *
+ * @package  automattic/jetpack-identity-crisis
+ */
+
+namespace Automattic\Jetpack\IdentityCrisis;
+
+/**
+ * Exception class for the Identity Crisis package.
+ */
+class Exception extends \Exception {}
```

Comment:
This class feels a bit redundant. Why do we need it?

Reply:
We use it to differentiate the IDC exceptions from all other kinds of exceptions we wouldn't want to accidentally handle ([here](https://github.com/Automattic/jetpack/blob/add/idc-generate-secret-endpoint/projects/packages/identity-crisis/src/class-rest-endpoints.php#L218) and [here](https://github.com/Automattic/jetpack/blob/add/idc-generate-secret-endpoint/projects/packages/identity-crisis/src/class-url-secret.php#L142)).